camviewer_ui_impl.py

```diff
@@ -578,14 +475,14 @@ def __init__(self, app, cwd, instrument, camera, cameraPv,
             idx = i
             break
         if idx >= 0:
-          print "Camera PV %s --> index %d" % (cameraPv, idx)
+          print("Camera PV %s --> index %d" % (cameraPv, idx))
```

Comment:
Assuming you used 2to3 for this, I wanted to point out pyupgrade as a nice "automated next step" for jumping to the Python 3 age: https://github.com/asottile/pyupgrade

Reply:
No, emacs. ;-)